components/properties/src/bidi_data.rs

```diff
@@ -154,8 +155,8 @@ impl<'a> BidiAuxiliaryPropertiesBorrowed<'a> {
         let mirroring_glyph = bidi_aux_props.mirroring_glyph;
         let paired_bracket_type = bidi_aux_props.paired_bracket_type;
         match paired_bracket_type {
-            BidiPairedBracketType::Open => BidiPairingProperties::Open(mirroring_glyph),
-            BidiPairedBracketType::Close => BidiPairingProperties::Close(mirroring_glyph),
+            CheckedBidiPairedBracketType::Open => BidiPairingProperties::Open(mirroring_glyph),
+            CheckedBidiPairedBracketType::Close => BidiPairingProperties::Close(mirroring_glyph),
```

Comment:
I think this is the place where you consume the `CheckedBidiPairedBracketType`, right? I don't understand the advantage of using the closed enum instead of using the open enum throughout.

Reply:
We can't use the open enum throughout: it has states that cannot be represented in the ULE type because we don't have space.

If we want to use the open enum, `MirroredPairedBracketData` needs to have private fields, with a custom validating serde impl (and a custom ctor for databake).